README.md

```diff
@@ -1,29 +1,134 @@
+</p>
 
-Wrapper around the Sauce Labs REST API for [Node.js](http://nodejs.org/).
-
-> **NOTE:**. 
->
-> This module was originally created by [Dan Jenkins](https://github.com/danjenkins). 
-With the help of multiple contributors ([Daniel Perez Alvarez](https://github.com/unindented), [Mathieu Sabourin](https://github.com/OniOni), [Michael J Feher](https://github.com/PhearZero), and many more) it became the module it is now.
->
-> On the 18th of February 2019 Sauce Labs has been given the ownership and maintenance of this module and will enhance it in the near feature with support for:
-> - all new API features
-> - support for the US and European Data Center
-> - Real device cloud support
-> - and many more
->
-> We would like to thank Dan and all contributors for their support and this beautiful module.
-
-![Trophy bot](./assets/trophy-bot.png)    
+# Node Sauce Labs [![Build Status](https://travis-ci.org/saucelabs/node-saucelabs.svg?branch=master)](https://travis-ci.org/saucelabs/node-saucelabs)
 
+Wrapper around the Sauce Labs REST APIs for [Node.js](http://nodejs.org/) (v8 or higher).
 
 ## Install
 
+To install the package run:
+
 ```shell
 npm install saucelabs
 ```
 
+## Options
+
+### user
+
+Your Sauce Labs username.
+
+Type: `string`<br>
+Default: `process.env.SAUCE_USERNAME`
+
+### key
+
+Your Sauce Labs access key.
+
+Type: `string`<br>
+Default: `process.env.SAUCE_ACCESS_KEY`
+
+### region
+
+Your Sauce Labs datacenter region. The following regions are available:
+
+- `us-west-1` (short `us`)
+- `eu-central-1` (short `eu`)
+
+Type: `string`<br>
+Default: `us`
+
+### headless
+
+If set to true you are accessing the headless Sauce instances (this discards the `region` option).
+
+Type: `boolean`<br>
+Default: `false`
+
+## Usage
+
+All accessible API commands with descriptions can be found [here](docs/interface.md).
+
+### As CLI Tool
+
+This package if installed globally can be used as CLI tool to access the API from the command line:
+
+```sh
+$ npm install -g saucelabs
+...
+$ sl listJobs $SAUCE_USERNAME --limit 5 --region eu
+{ jobs:
+   [ { id: '19dab74f8fd848518f8d2c2cee3a6fbd' },
+     { id: 'dc08ca0c7fa14eee909a093d11567328' },
+     { id: '5bc6f70c777b4ae3bf7909a40f5ee41b' },
+     { id: 'f40fe7b044754eaaa5f5a130406549b5' },
+     { id: 'd1553f71f910402893f1e82a4dcb6ca6' } ] }
+```
+
+You can find all available commands and options with description by calling:
+
+```sh
+sl --help
+# show description for specific command
+sl listJobs --help
+```
+
+### As NPM Package
+
+The following example shows how to access details of the last job you were running with your account that is being exposed as environment variables as `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`.
+
+```js
+import SauceLabs from 'saucelabs';
+
+(async () => {
+    const myAccount = new SauceLabs();
+
+    // get job details of last run job
+    const job = await user.listJobs(
+        process.env.SAUCE_USERNAME,
```

Comment:
Why do we need to provide a username here? It's already defaulted if not provided?

Reply:
the `listJobs` expects a username as first parameter because you could potentially also fetch the jobs of a sub account